homework_8/participants/Main.java

```diff
@@ -0,0 +1,25 @@
+package participants;
+
+public class Main {
+    public static void main(String[] args) {
+        Participant[] participants = {
+                new Human("John", 15, 0.5),
+                new Cat("Iris", 20, 1),
+                new Robot("GH-1782", 30, 0.7),
+                new Human("Bella", 25, 1.1)
+        };
+        Barrier[] barriers = {
+                new Racetrack("Race-track #1", 20),
+                new Wall("Wall #1", 1),
+                new Racetrack("Race-track #2", 25)
+        };
+
+        for (Participant part : participants) {
```

Comment:
It looks like a competition, isn't it? Create a new class `Competition` and move this logic to a method `start`